lib/adyen/errors.rb

```diff
@@ -1,6 +1,6 @@
 module Adyen
   class AdyenError < StandardError
-    attr_reader :code, :response, :request
+    attr_reader :code, :response, :request, :msg
```

Comment:
could we have a `message` method containing a summary of all the properties ?
```
    def message
      attributes = {code: code, response: response, request: request, msg: msg}.compact.map{ |k, v| "#{k}:#{v}"}.join(", ")
      "#{self.class.name} #{attributes}"
    end
```
and 
```
Adyen::AuthenticationError.new("message", "request").message
```
would render
```
"Adyen::AuthenticationError code:401, request:request, msg:message"
```
which would definitely be helpful

Reply:
Hi Xavier,

Sure, I have no problem with adding a convenience method to access these properties.  Can you submit a pull request?

Reply:
Thanks @crrood .

I've created the PR: https://github.com/Adyen/adyen-ruby-api-library/pull/27